app/routes/admin/index.js

```diff
@@ -1,6 +1,24 @@
 import Ember from 'ember';
 
-const { Route } = Ember;
+const { Route, RSVP } = Ember;
 
 export default Route.extend({
+  model() {
+    return RSVP.hash({
+      events: this.get('store').queryRecord('admin-statistics-event', {
+        filter: {
+          name : 'id',
+          op   : 'eq',
+          val  : 1
```

Comment:
@sumedh123 The data is at id 1.
Please check https://open-event-api.herokuapp.com/#admin-statistics